service/settings.toml

```
@@ -5,3 +5,5 @@ DATABASE_URL = "sqlite:///:memory:"
 SEARCH_QUERY= 'topic:"flask-extension flask-extensions"'
```

Parece que é nesta linha: https://github.com/BurntSushi/toml-test/blob/master/tests/valid/string-escapes.toml

```suggestion
SEARCH_QUERY= "topic:'flask-extension flask-extensions'"
```

---

A CLI cria novamente o **GITHUB_TOKEN** 

> 
> cd service  
> sed -i 's/blocking=True/blocking=False/g' fexservice/cli.py
> echo "GITHUB_TOKEN = \"$GITHUB_TOKEN\"" >> settings.toml  
> poetry run fexservice  
> shell: /bin/bash -e {0}  
> env:  
> pythonLocation: /opt/hostedtoolcache/Python/3.8.2/x64  
> GITHUB_TOKEN: 
  
O erro é de dados duplicados _ValueError: Duplicate keys!_
  
> Traceback (most recent call last):
>   File "/home/runner/.virtualenvs/fexservice-2WvbHRfJ-py3.8/lib/python3.8/site-packages/toml/decoder.py", line 455, in loads
>     ret = decoder.load_line(line, currentlevel, multikey,
>   File "/home/runner/.virtualenvs/fexservice-2WvbHRfJ-py3.8/lib/python3.8/site-packages/toml/decoder.py", line 728, in load_line
>     raise ValueError("Duplicate keys!")
> ValueError: Duplicate keys!

Acho que o melhor é remover o **GITHUB_TOKEN** do _settings.toml_